cpu/stm32/kconfigs/Kconfig.clk

```diff
@@ -65,15 +65,68 @@ config CLOCK_PLL_M
 
 config CLOCK_PLL_N
     int "N: PLLIN multiply factor" if CUSTOM_PLL_PARAMS
+    default 120 if BOARD_HAS_HSE && CPU_FAM_F2
+    default 60 if CPU_FAM_F2
+    default 168 if CPU_FAM_F4 && CLOCK_MAX_84MHZ && BOARD_HAS_HSE
+    default 84 if CPU_FAM_F4 && CLOCK_MAX_84MHZ
+    default 96 if CPU_FAM_F4 && CLOCK_MAX_100MHZ && BOARD_HAS_HSE && MODULE_PERIPH_USBDEV
+    default 48 if CPU_FAM_F4 && CLOCK_MAX_100MHZ && MODULE_PERIPH_USBDEV
+    default 100 if CPU_FAM_F4 && CLOCK_MAX_100MHZ && BOARD_HAS_HSE
+    default 50 if CPU_FAM_F4 && CLOCK_MAX_100MHZ
+    default 168 if CPU_FAM_F4 && CLOCK_MAX_180MHZ && BOARD_HAS_HSE && MODULE_PERIPH_USBDEV
```

Comment:
For some reason the `MODULE_PERIPH_USBDEV` case is not taken into account. @leandrolanzieri is it possible to have a conditional at this level on the presence of a module ?

Reply:
I think you will need to use `&& (MODULE_PERIPH_USBDEV || USEMODULE_PERIPH_USBDEV)`, the first one is the module in the migration and the second one is the injected from `USEMODULE`.

Reply:
It works and I found another issue with the stm32f429i-disco. It should work ok now